apps/mull-api/src/app/event/event.resolver.spec.ts

```diff
@@ -3,13 +3,19 @@ import { EventResolver } from './event.resolver';
 import { EventService } from './event.service';
 import { CreateEventInput, UpdateEventInput } from './inputs/event.input';
 import { mockPartialEvent, mockAllEvents } from './event.mockdata';
+import { User } from '../entities';
 
 const mockEventService = () => ({
   create: jest.fn((mockEventData: CreateEventInput) => ({ ...mockEventData })),
   findOne: jest.fn((id: number) => mockAllEvents.find((event) => event.id === id)),
   findAll: jest.fn(() => mockAllEvents),
   update: jest.fn((mockEventData: UpdateEventInput) => ({ ...mockEventData })),
   delete: jest.fn((id: number) => mockAllEvents.find((event) => event.id === id)),
+  addParticipant: jest.fn((eventId: number, userId: number) => {
+    const event = mockAllEvents.find((event) => (event.id = eventId));
+    event.participants.push(new User(userId));
```

Comment:
Just a question, does `(new User(userId))` create a new user in the database? Or is it like a `find_or_create` ?